python/ray/services.py

```diff
@@ -559,12 +556,10 @@ def wait_for_redis_to_start(redis_ip_address,
             redis_client.client_list()
         except redis.ConnectionError:
             # Wait a little bit.
-            time.sleep(1)
```

Comment:
Also, this `time.sleep` was probably making `ray.init()` a second slower than it needed to be.